uk_bin_collection/uk_bin_collection/councils/NeathPortTalbotCouncil.py

```
@@ -30,14 +30,18 @@ def parse_data(self, page: str, **kwargs) -> dict:
             # Accept cookies banner
-            cookieAccept = WebDriverWait(driver, 10).until(
-                EC.presence_of_element_located((By.ID, "ccc-notify-accept"))
-            )
-            cookieAccept.click()
+            try:
+                cookieAccept = WebDriverWait(driver, 10).until(
+                    EC.presence_of_element_located((By.ID, "ccc-notify-accept"))
+                )
+                driver.execute_script("arguments[0].click();", cookieAccept)
+            except Exception:
+                pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)